### DISCUSSION TITLE
[Dashboard tabs] UX for upper tab is confusing

### DISCUSSION THREAD

**Discussion by rosemarie-chiu** [apache/superset#19191] [Ideas]
**Is your feature request related to a problem? Please describe.**
I like that we have upper tabs, but the UX for it is very confusing to users
1. If I already have content on my dashboard, THEN I add an upper tab component, it isn't obvious to me that the current components ARE on this upper tab until I add second upper tab and switch tabs.
![CleanShot 2021-07-15 at 12 04 06](https://user-images.githubusercontent.com/52086618/125843360-d4f529f5-be98-4276-a1bd-71846f7e0a09.gif)

2. In dashboard edit mode, it is very hard to switch upper tab. I keep getting edit tab name instead of switching tab. Have to click on precisely that tab's bottom line to switch tab. Not very ideal.

https://user-images.githubusercontent.com/52086618/125843821-ca2200a0-cc09-45a2-9d28-023a6b62d805.mp4

**Describe the solution you'd like**
I'm suggesting instead of making upper tab a drag and drop component like inner tab does, we can do it like Safari. when you first launch it, it show a small tab icon on the very right. You can add tabs according to your need?

https://user-images.githubusercontent.com/52086618/125844790-9cc2d0a8-4fb4-4caf-9aa9-5dd6918ff158.mp4

**Describe alternatives you've considered**
Or, we show upper tab up front when user create a new dashboard. And if the user doesn't need tab, they can collapse it to not show?
For example, whenever you open chrome browser, tabs is already suggested. It is clear to me that whatever I'm doing right now, it is on the first tab and I have option to create more.
![CleanShot 2021-07-15 at 12 13 37](https://user-images.githubusercontent.com/52086618/125844432-ca586ef9-83cb-48c6-a7ff-a248e36c1785.png)

**Comment by a-cid** [comment]
I can't add upper tabs at all. My setup is running 1.2 installed from pip, upgraded from 1.0.1, it worked fine before upgrading.

https://user-images.githubusercontent.com/834150/129401312-523252f0-305d-488b-a64e-13191f6aabb9.mp4

**Comment by junlincc** [comment]
@michael-s-molina 🙏
